Mage.Sets/src/mage/cards/l/LazierGoblin.java

```diff
@@ -0,0 +1,191 @@
+        super(Outcome.Benefit);
+    }
+
+    private LazierGoblinMotivateEffect(final LazierGoblinMotivateEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public LazierGoblinMotivateEffect copy() {
+        return new LazierGoblinMotivateEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        LazierGoblinWatcher watcher = game.getState().getWatcher(LazierGoblinWatcher.class);
+        Permanent permanent = source.getSourcePermanentOrLKI(game);
+        if (watcher == null || permanent == null) {
+            return false;
+        }
+        watcher.motivationPaid(source.getControllerId(), new MageObjectReference(permanent, game));
+        return true;
+    }
+}
+
+class LazierGoblinRestrictionEffect extends RestrictionEffect {
+
+    public LazierGoblinRestrictionEffect() {
+        super(Duration.WhileOnBattlefield);
+        staticText = ""; // no text.
+    }
+
+    private LazierGoblinRestrictionEffect(final LazierGoblinRestrictionEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public LazierGoblinRestrictionEffect copy() {
+        return new LazierGoblinRestrictionEffect(this);
+    }
+
+    @Override
+    public boolean applies(Permanent permanent, Ability source, Game game) {
+        LazierGoblinWatcher watcher = game.getState().getWatcher(LazierGoblinWatcher.class);
+
+        return watcher != null
+                && permanent != null
+                && permanent.getId().equals(source.getSourceId())
+                && !watcher.isMotivated(permanent.getControllerId(), new MageObjectReference(permanent, game));
+    }
+
+    @Override
+    public boolean canBlock(Permanent attacker, Permanent blocker, Ability source, Game game, boolean canUseChooseDialogs) {
+        return false;
+    }
+
+    @Override
+    public boolean canAttack(Game game, boolean canUseChooseDialogs) {
+        return false;
+    }
+}
+
+enum LazierGoblinMotivatedCondition implements Condition {
+    instance;
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        LazierGoblinWatcher watcher = game.getState().getWatcher(LazierGoblinWatcher.class);
+        Permanent permanent = game.getPermanentOrLKIBattlefield(source.getSourceId());
+
+        return watcher != null
+                && permanent != null
+                && watcher.isMotivated(permanent.getControllerId(), new MageObjectReference(permanent, game));
+    }
+
+    @Override
+    public String toString() {
+        return "motivated";
+    }
+}
+
+class LazierGoblinWatcher extends Watcher {
+
+    // Player -> all Lazier Goblin MOR that were motivated by that player.
+    private final Map<UUID, Set<MageObjectReference>> motivationMap = new HashMap<>();
+
+    public LazierGoblinWatcher() {
+        super(WatcherScope.GAME);
+    }
+
+    @Override
+    public void watch(GameEvent event, Game game) {
+        // This watcher does not watch anything. Shall it be set up another way?
```

Comment:
other cards with special actions that affect restriction effects store values directly in the game state (one messy example: Lost in Thought)

not sure this is actually wrong though, take a look and see what you think is cleaner

Reply:
Yeah I did consider that option. A watcher that is only a map storage is weird enough I'll rework with a direct storage.